sound/soc/sof/ipc4-topology.c

```diff
@@ -530,10 +531,12 @@ static int sof_ipc4_widget_setup_comp_dai(struct snd_sof_widget *swidget)
 
 	pipe_widget = swidget->spipe->pipe_widget;
 	pipeline = pipe_widget->private;
-	if (pipeline->use_chain_dma && ipc4_copier->dai_type != SOF_DAI_INTEL_HDA) {
-		dev_err(scomp->dev,
-			"Bad DAI type '%d', Chained DMA is only supported by HDA DAIs (%d).\n",
-			ipc4_copier->dai_type, SOF_DAI_INTEL_HDA);
+
+	/* chain_dma is available for all DAIs on dspless platforms */
```

Comment:
This is not true, TGL is 'dspless platform' but only HDA dais can support DSPless mode.

In comments we tend to use "DSPless"

Reply:
removed comment